resources/assets/components/pages/VoterRegistrationDrivePage/Beta/StartVoterRegistrationForm.js

```diff
@@ -18,18 +18,17 @@ const StartVoterRegistrationForm = ({ campaignId, referrerUserId }) => {
     return name === 'email' ? setEmail(value) : setZip(value);
   };
 
-  const handleClick = () => {
-    if (!isDisabled) {
-      trackAnalyticsEvent('click_voter_registration_action', {
-        action: 'button_clicked',
-        category: EVENT_CATEGORIES.campaignAction,
-        label: 'voter_registration',
-        context: {
-          campaignId,
-        },
-      });
-      window.location = `https://register.rockthevote.com/registrants/new?partner=37187&source=user:${referrerUserId},source:web,source_details:onlinedrivereferral,referral=true&email_address=${email}&home_zip_code=${zip}`;
-    }
+  const handleSubmit = event => {
+    event.preventDefault();
+    trackAnalyticsEvent('click_voter_registration_action', {
```

Comment:
Nit - I think new lines before and after this `trackAnalyticsEvent` function call would improve readability a smidge.

Reply:
Agreed! I ❤️ carriage returns and room to breeeeathe.

Reply:
Should this event name be past-tense?
```suggestion
    trackAnalyticsEvent('clicked_voter_registration_action', {
```

Reply:
@mendelB 👀 🦅 !! You are right:

https://github.com/DoSomething/phoenix-next/blob/15583dccdbee7c42621832bb04e933066cf8a1c0/resources/assets/components/actions/VoterRegistrationAction/VoterRegistrationAction.js#L40

We ended up creating a separate component vs adding this into the `VoterRegistrationAction` like we had talked about in a past megagroom - because it was too much overhead to try to break the existing component `VoterRegistrationAction` into templates for our one-time use.